modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -167,6 +169,26 @@ class Instrument_List extends \NDB_Menu_Filter
             $prevSubgroup = null;
 
             foreach ($listOfInstruments as $instrument) {
+                $isSurvey = $DB->pselectOne(
+                    "SELECT IsDirectEntry
+                    FROM test_names
+                    WHERE Test_name=:TN",
+                    array(
+                     'TN' => $instrument['Test_name'],
+                    )
+                );
+
+                $isSent = $DB->pselectOne(
+                    "SELECT ID
+                    FROM participant_accounts
+                    WHERE Test_name=:TN
+                    AND CommentID=:CID",
+                    array(
+                     'TN'  => $instrument['Test_name'],
+                     'CID' => $instrument['CommentID'],
+                    )
+                );
+
                 // print the sub group header row
                 if ($instrument['Sub_group'] != $prevSubgroup) {
```

Comment:
given that `$instrument['Sub_group'] ` is set to `$prevSubgroup` I think you can add an extra equal sign (`!==`) for a type check as well.

Reply:
Sorry, just realised this change was not made by you. Ignore that!